engine/engine/src/engine.cpp

```diff
@@ -849,6 +857,15 @@ namespace dmEngine
         // This scope is mainly here to make sure the "Main" scope is created first
         DM_PROFILE("Init");
 
+        char window_title[512];
+        const char* project_title = dmConfigFile::GetString(engine->m_Config, "project.title", "TestTitle");
+#if !defined(DM_RELEASE)
+        if (instance_index)
+        {
+            dmSnPrintf(window_title, sizeof(window_title), "%s - %d", project_title, instance_index);
```

Comment:
add index into the title. It's convenient, but also it gives each windows position save on mac for free (it's OS functionality we use now in glfw3)